commands/slowlog.md

```diff
@@ -34,7 +34,8 @@ parameter to zero) with minor performance hit.
 To read the slow log the **SLOWLOG GET** command is used, that returns every
 entry in the slow log.
 It is possible to return only the N most recent entries passing an additional
-argument to the command (for instance **SLOWLOG GET 10**).
+argument to the command (for instance **SLOWLOG GET 10**). It is also allowed
+to pass in -1 to get all slow logs (for instance **SLOWLOG GET -1**).
```

Comment:
please have a look at the styling guidelines: https://github.com/redis/redis-doc/blob/master/README.md
to avoid mess in git log and easier review, from now on, we put each sentence in it's own line (making an effort to only change ones we actually change).
```suggestion
argument to the command (for instance **SLOWLOG GET 10**).
The default requested length is 10 (when the argument is omitted), It is possible to pass -1 for getting the entire slowlog.
```
@itamarhaber please review the wording and fix.

Reply:
Make it:
`The default requested length is 10 (when the argument is omitted). It's possible to pass -1 to get the entire slowlog.`